Dockerfile

```
@@ -0,0 +1,20 @@
+# Dockerfile
+FROM docker.io/library/gradle:7.5.1-jdk11 AS build
+
+WORKDIR /app
+
+# add project
+ADD . /app/
+
+RUN gradle clean bootJar
+
+FROM docker.io/library/eclipse-temurin:11-jre-focal AS app
+
+WORKDIR /app
+COPY --from=build /app/build/libs/spring-petclinic-2.7.3.jar .
```

In my opinion, it would be better not to specify the version.
```suggestion
COPY --from=build /app/build/libs/spring-petclinic-*.jar spring.petclinic.jar
```